Index: tools/epydoc.sh
===================================================================
--- tools/epydoc.sh.orig
+++ tools/epydoc.sh
@@ -7,22 +7,12 @@
 
 name=pysilhouette
 PATH=${PATH}:/usr/bin
-export PYTHONPATH=${PYTHONPATH}:/usr/lib/python:/usr/lib/python2.6:/usr/lib/python2.6/site-packages
+export PYTHONPATH=${PYTHONPATH}:/usr/lib/python:@py_dir@:@python_sitelib@
 
-script_dir=`dirname $0`
-pushd $script_dir >/dev/null 2>&1
-# shell directory.
-script_dir=`pwd`
-popd >/dev/null 2>&1
+epydoc_config=@docdir@/python-${name}-doc/epydoc.cfg
+target=@ap_docroot@/${name}-doc
 
-epydoc_config=${script_dir}/../doc/epydoc.cfg
+[ -e ${target} ] &&  rm -rf ${target} ;
+install -d ${target}
 
-
-target=/var/www/html/${name}-doc
-
-if [ -e ${target} ]; then
-  rm -fr ${target}
-fi
-mkdir -p ${target}
-
-epydoc --config ${epydoc_config}
+epydoc -v --config ${epydoc_config}
