Index: sample/silhouette.conf.example
===================================================================
--- sample/silhouette.conf.example.orig
+++ sample/silhouette.conf.example
@@ -5,7 +5,7 @@ env.python=/usr/bin/python
 env.sys.log.conf.path=/etc/pysilhouette/log.conf
 # To set a unique key, please.
 #     command : python uniqkey.py or uuidgen
-env.uniqkey=2f21b1be-a132-415b-aab6-03cccecb6d7c
+env.uniqkey=@uniqkey@
 
 ##
 # daemon
@@ -16,10 +16,10 @@ daemon.stderr=/var/log/pysilhouette/stde
 ##
 # observer
 observer.target.python=/usr/bin/python
-observer.target.scheduler=/usr/lib/python2.6/site-packages/pysilhouette/scheduler.py
-observer.target.performer=/usr/lib/python2.6/site-packages/pysilhouette/performer.py
-observer.target.asynscheduler=/usr/lib/python2.6/site-packages/pysilhouette/asynscheduler.py
-observer.target.asynperformer=/usr/lib/python2.6/site-packages/pysilhouette/asynperformer.py
+observer.target.scheduler=@python_sitelib@/pysilhouette/scheduler.py
+observer.target.performer=@python_sitelib@/pysilhouette/performer.py
+observer.target.asynscheduler=@python_sitelib@/pysilhouette/asynscheduler.py
+observer.target.asynperformer=@python_sitelib@/pysilhouette/asynperformer.py
 observer.restart.count=5
 #  - Clear intervals. 0=Infinite
 observer.restart.count.clear.time=600
@@ -30,22 +30,22 @@ observer.status.path=/var/lib/pysilhouet
 
 ##
 # performer
-performer.mkfifo.group.name=pysilhouette
+performer.mkfifo.group.name=psil
 performer.mkfifo.stop.code=2
 performer.mkfifo.ignore.code=1
 performer.mkfifo.path=/tmp/pysilhouette-performer.fifo
 performer.mkfifo.start.code=0
-performer.mkfifo.user.name=pysilhouette
+performer.mkfifo.user.name=psil
 performer.mkfifo.perms=0666
 
 ##
 # asynperformer
-asynperformer.mkfifo.group.name=pysilhouette
+asynperformer.mkfifo.group.name=psil
 asynperformer.mkfifo.stop.code=2
 asynperformer.mkfifo.ignore.code=1
 asynperformer.mkfifo.path=/tmp/pysilhouette-asynperformer.fifo
 asynperformer.mkfifo.start.code=0
-asynperformer.mkfifo.user.name=pysilhouette
+asynperformer.mkfifo.user.name=psil
 asynperformer.mkfifo.perms=0666
 asynscheduler.interval=10
 asynperformer.thread.pool.size=5
